org.gnome.Games.json

```diff
@@ -204,8 +204,8 @@
                 {
                     "type" : "git",
                     "url" : "https://gitlab.gnome.org/GNOME/gnome-games.git",
-                    "tag" : "3.34.0",
-                    "commit" : "d6e511a9b9d451ce6755fff19390ae9880585896"
+                    "branch" : "gnome-3-34",
```

Comment:
Can't build a branch, use a specific tag.

Reply:
Hm, I have a specific commit though, is that not enough? There's no tag, I wanted to push an update in advance, since it's not 3.34.1 time yet, like Builder does.

Reply:
You can use a specific commit of course. but currently you have a branch name that is constantly changing which conflicts with the commit. Just remove that then.

Reply:
Ah, ok. :+1:

Reply:
I admit I haven't seen the actual build error, since it couldn't fetch shared modules.